apiserver/plane/utils/issue_filters.py

```
@@ -622,6 +622,89 @@ def filter_issue_type_id(params, issue_filter, method, prefix=""):
     if type_id is not None:
         issue_filter[f"{prefix}type_id"] = type_id
 
+
+def timestamp_filter(issue_filter, field, queries):
+    """
+    Applies timestamp filtering on the specified field using a list of query strings.
+
+    Each query should be in the format:
+      <ISO8601_timestamp>[;operator]
+
+    - The timestamp must be in ISO 8601 format with millisecond precision (e.g., 2024-06-01T12:00:00.000Z).
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)